db/seeds/index.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Movies    = require('./data/movies');
+const Locations = require('./data/locations');
 
-exports.seed = async (Knex) => {
-  await Knex('movies').truncate();
-  await Knex('movies').insert(Movies);
+exports.seed = (Knex) => {
```

Comment:
I think you can make this seed function just one large `async` function and then call `await` in 3 consecutive lines to eliminate the need for a `return` and to get rid of the promise chain.

Reply:
Are there any specific advantages/disadvantages in your mind?
So far I was able find these -
* https://hackernoon.com/should-i-use-promises-or-async-await-126ab5c98789
* https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9

Reply:
The main reasons why I tend to defer to `async/await` is because it's easier to read and conceptualize in my head and because I think as a company, we've tended to use `async/await` during recent times rather than promises.